wemake_python_styleguide/violations/naming.py

```diff
@@ -779,3 +781,39 @@ class WrongUnusedVariableNameViolation(ASTViolation):
 
     error_template = 'Found wrong unused variable name: {0}'
     code = 123
+
+
+@final
+class UnreadableNameViolation(MaybeASTViolation):
+    """
+    Forbids to have variable or module names which could be difficult to read.
+
+    Reasoning:
+        Currently one can name your classes like so: `ControlIn`
+        Inside it is just L and i, but we cannot tell it from the word.
+        There are a lot other combinations which are unreadable
+
+    Solution:
+        We need to forbid unreadable letter combinations.
+
+    This rule checks: modules, variables, attributes,
```

Comment:
We usually put this before `Reasoning`

Reply:
You also have to add a link to the constant with all letters. See how we use `:py:data:` for this.

Reply:
> We usually put this before `Reasoning`

My code is based on examples I found: https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/violations/naming.py#L328

Reply:
Ok! I would later need to fix that by adding explicit paragraph ordering. Thanks for the find 👍